src/modules/auth/passwd/ResetPwdScreen.js

```diff
@@ -27,14 +35,52 @@ function ResetPwdScreen(props) {
 export default ResetPwdScreen;
 
 function ResetPwdCard(props) {
-	const { t, i18n } = useTranslation();
-	const { handleSubmit, register, getValues, formState: { errors, isSubmitting } } = useForm();
+	const { t } = useTranslation();
+	const SeaCatAuthAPI = props.app.axiosCreate('seacat-auth');
+	const { handleSubmit, register, getValues, watch, formState: { errors, isSubmitting } } = useForm();
 
 	generatePenrose();
 	
 	let history = useHistory();
```

Comment:
@byewokko `const`

Reply:
i know, there is plenty of eslint issues, but that's for another time :grin: